src/MdProcessor.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Learn2Blog.src
+{
+    public partial class MdProcessor
+    {
+        /* public static string ConvertMdToHtml(string title, string text)
```

Comment:
remove this comment block before the final merge. but I like suggested, remove this class and .cs file altogether and add the new function below to `HtmlProcessor` class.